Resources/Prototypes/_NF/XenoArch/Effects/utility_effects.yml

```diff
@@ -0,0 +1,110 @@
+# Utility effects permanently modify the entity in some way when triggered, and they generally make it 'useful' for some purpose,
+# like turning the artifact into a tool, or gun, or whatever.
+- type: artifactEffect
+  id: EffectReclaimer
+  targetDepth: 3
+  effectHint: artifact-effect-hint-multitool
+  blacklist:
+    components:
+    - Item
+  permanentComponents:
+  - type: MaterialReclaimer
```

Comment:
I will need to play around with this a bit, but I expect this will cause confusion and problems as the artifact randomly destroys things. It is not obvious that the thing is a material reclaimer. I know a lot of artifact effects aren't obvious, but... ehhh. I'm not super stoked about this one.

I repeat my question: Why?

Reply:
Funny as hell.

Reply:
> It is not obvious that the thing is a material reclaimer

- Pokes artefact with wrench
- Wrench disappears
- "Material reclaimer noises"
- A single sheet of metal appears.
- Wonder what could it be.